src/Simulation/Simulators.Type1/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// Allow the simulator assembly to use our internal methods
+[assembly: InternalsVisibleTo("Tests.Microsoft.Quantum.Simulators.Type1" + SigningConstants.PUBLIC_KEY)]
```

Comment:
This one is a test assembly. :) Although personally I think the comment is kind of redundant, so I don't mind if it's removed either.

Reply:
Oops! My find replace got a little too aggressive...